staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -158,7 +158,7 @@ func (c *APIServiceRegistrationController) processNextWorkItem() bool {
 func (c *APIServiceRegistrationController) enqueueInternal(obj *v1.APIService) {
 	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
-		klog.Errorf("Couldn't get key for object %#v: %v", obj, err)
+		klog.ErrorS(err, "Couldn't get key for object", "object", obj)
```

Comment:
What will this emit? The existing line emits type information that would be useful for debugging.

Reply:
Info on `ErrorS` from the migration guide: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes

Reply:
Please correct me if you disagree Daniel, but I think your asking for an additional field in the structured output which is the type? The # automatically adding that information to the old logged output. https://play.golang.org/p/ULYQ34hInsK shows the difference between ```%#v``` and ```%v``` and how that additional field could be obtained.

Reply:
Yeah, that's what I was thinking. Although I now note that despite the generic name and `%#v`, 'obj' is actually strongly typed so it isn't actually important.